js/app.js

```diff
@@ -221,10 +222,20 @@ function buildContentSnippet(blogSummary, data, key) {
         const snippetContent = firstOrDefaultValue(data[key], snippetType.property) || "";
         if (snippetType.format === "text") elementSnippet.textContent = snippetContent;
         else elementSnippet.innerHTML = snippetContent;
+        if (snippetType.classname) elementSnippet.classList.add(snippetType.classname);
         if (typeCode === "aside_note") {
             const asideStyle = firstOrDefaultValue(data[key], "note_style");
             if (asideStyle) elementSnippet.classList.add("aside-" + asideStyle.codename);
         }
+        else if (typeCode === "quotation") {
+            const quoteReference = firstOrDefaultValue(data[key], "reference");
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite is a good example of using semantic HTML to express things like this.

Reply:
Yes, I thought of that. However, the reference field itself contains rich text, which in theory could contain various different HTML elements. As an editorial guide we could restrict this to allow one hyperlink, which must contain the title of the cited creative work (as per the definition). We can then wrap the hyperlink in a <cite> block.

Reply:
Actually it looks like the meaning of <cite> has changed in HTML5:

The cite element represents the title of a work (e.g. a book, a paper, an essay, a poem, a score, a song, a script, a film, a TV show, a game, a sculpture, a painting, a theatre production, a play, an opera, a musical, an exhibition, a legal case report, etc). This can be a work that is being quoted or referenced in detail (i.e. a citation), or it can just be a work that is mentioned in passing.

Reply:
However both `q` and `blockquote` appear to be candidates for here.

Reply:
OK, used `blockquote `and footer as per the example on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite but left `<cite>` out for now